### ISSUE TITLE
0.61.5 Discussion

### ISSUE THREAD

**Issue by grabbou** [react-native-community/releases#154]
Conversations on this thread are limited to 0.61 releases major issues and backport (cherry-pick) requests from commits that are already on master.

An example of a good such request is a bug fix for a serious issue that has been merged into master but did not make the 0.61.5 cut.

In other words, if you cannot point to a particular commit on master, then your request likely belongs as a new issue in http://github.com/facebook/react-native/issues.

**Comment by sibelius** [comment]
can we pick the new cli version?

https://github.com/react-native-community/cli/pull/768

to improve support to monorepos on android?

**Comment by kelset** [comment]
@sibelius you don't need to wait for a new release for that, you can use `yarn resolutions` in your package.json as such:

```
  "resolutions": {
    "@react-native-community/cli-platform-android": "3.0.3",
    "@react-native-community/cli-platform-ios": "3.0.0",
    "@react-native-community/cli-tools": "3.0.0",
    "@react-native-community/cli-types": "3.0.0",
    "@react-native-community/cli": "3.0.4",
}
```

**Comment by grabbou** [comment]
You don't really need to do anything I guess. `react-native` depends on `^3.0.0`, so it will get all the latest CLIs anyway, without any resolutions needed.

**Comment by nar213** [comment]
Please can you consider looking into PR below to resolve Android children pointerEvent issues

**PR Request:** 
https://github.com/facebook/react-native/pull/26374

**Related Issues:** 
https://github.com/facebook/react-native/issues/26099
https://github.com/facebook/react-native/issues/27232
https://github.com/facebook/react-native/issues/26147

**Comment by Aure77** [comment]
For accessibility purpose on touchable elements, can you cherry-pick this commit : https://github.com/facebook/react-native/pull/27293/commits/68a10a57920d6ee8d0eb177730011c38b7664f03

PR: https://github.com/facebook/react-native/pull/27293

**Comment by nokite** [comment]
hermesCommand in react.gradle uses wrong path. That's when you use a custom folder structure and do a release build with Hermes enabled. The issue comes from this line:
https://github.com/facebook/react-native/blob/0e4bcaa2960a2b1aa42dbe716fc6a35652aa7207/react.gradle#L33
I opened a new issue, but I think it's also relevant to mention here, as I read that this issue was fixed in the past, and might be returning.
https://github.com/facebook/react-native/issues/28126
